plugins/services/src/js/types/Service.ts

```diff
@@ -0,0 +1,13 @@
+import { ServicePlan } from "#PLUGINS/services/src/js/types/ServicePlan";
+
+export interface Service {
+  name: string;
+  plans: ServicePlan[];
+}
+
+export const ServiceSchema = `
+type Service {
+  name: String!
+  plans(name: String = ""): [ServicePlan]!
```

Comment:
I don't think our implementation honors the default value for an argument, partly because I have never heard about that graphql feature, nice work 👍